CSS/CSS.sublime-syntax

```diff
@@ -820,46 +813,33 @@ contexts:
             3: punctuation.section.function.css
             4: constant.numeric.css
             5: punctuation.section.function.css
-        - match: '(?i)(\[)\s*(-?[_a-z\\[^[:ascii:]]][_a-z0-9\-\\[^[:ascii:]]]*)(?:\s*([~|^$*]?=)\s*(?:(-?[_a-z\\[^[:ascii:]]][_a-z0-9\-\\[^[:ascii:]]]*)|((")(?:[^\\]|\\.)*?("))))?\s*(\])'
-          scope: meta.attribute-selector.css
-          captures:
-            1: punctuation.definition.entity.css
-            2: entity.other.attribute-name.attribute.css
-            3: punctuation.separator.operator.css
-            4: string.unquoted.attribute-value.css
-            5: string.quoted.double.attribute-value.css
-            6: punctuation.definition.string.begin.css
-            7: punctuation.definition.string.end.css
-        - match: '(?i)(\[)\s*(-?[_a-z\\[^[:ascii:]]][_a-z0-9\-\\[^[:ascii:]]]*)(?:\s*([~|^$*]?=)\s*(?:(-?[_a-z\\[^[:ascii:]]][_a-z0-9\-\\[^[:ascii:]]]*)|(('')(?:[^\\]|\\.)*?(''))))?\s*(\])'
-          scope: meta.attribute-selector.css
-          captures:
-            1: punctuation.definition.entity.css
-            2: entity.other.attribute-name.attribute.css
-            3: punctuation.separator.operator.css
-            4: string.unquoted.attribute-value.css
-            5: string.quoted.single.attribute-value.css
-            6: punctuation.definition.string.begin.css
-            7: punctuation.definition.string.end.css
-  string-double:
-    - match: '"'
-      scope: punctuation.definition.string.begin.css
-      push:
-        - meta_scope: string.quoted.double.css
-        - match: '"'
-          scope: punctuation.definition.string.end.css
-          pop: true
-        - match: \\.
-          scope: constant.character.escape.css
-  string-single:
-    - match: "'"
-      scope: punctuation.definition.string.begin.css
-      push:
-        - meta_scope: string.quoted.single.css
-        - match: "'"
-          scope: punctuation.definition.string.end.css
-          pop: true
-        - match: \\.
-          scope: constant.character.escape.css
+        # Attribute Selectors
+        # https://drafts.csswg.org/selectors-4/#attribute-selectors
+        - match: '\['
+          scope: punctuation.definition.entity.css
+          push:
+            - meta_scope: meta.attribute-selector.css
+            - match: '(?:(-?[_a-z][_a-z0-9-]*)|(\*))?([|])(?!=)'
+              captures:
+                1: entity.other.namespace-prefix.css
+                2: entity.name.namespace.wildcard.css
+                3: punctuation.separator.namespace.css
+            - match: '[^\\ =<>`~*|^$\]\[]+'
+              scope: entity.other.attribute-name.css
+            - match: '\s*([~*|^$]?=)\s*'
+              captures:
+                1: keyword.operator.attribute-selector.css
+              push:
+                - meta_content_scope: entity.other.attribute-value.css
```

Comment:
`entity.other` doesn't feel like the correct scope to be applied here. The HTML syntax uses `string.quoted.double`, `string.quoted.single` or `string.unquoted`. I think we'd want to use that here also.